### DISCUSSION TITLE
福建平潭2个

### DISCUSSION THREAD

**Discussion by naclhhn** [WeNeedHome/SummaryOfLoanSuspension#949] [General]
![mmexport1658318073152](https://user-images.githubusercontent.com/14798592/179975724-8c5fbe70-d113-423d-bd49-c0cd8f79171b.jpg)
![mmexport1658318080746](https://user-images.githubusercontent.com/14798592/179975731-461d2749-5d69-4731-adce-694e03205347.jpg)

**Comment by super-aviator** [comment]
这项目也烂尾了
